integration-tests/rita.py

```diff
@@ -312,24 +312,20 @@ def switch_binaries(node_id):
             "BOUNTY_HUNTER:\t{}").format(RITA, RITA_EXIT, BOUNTY_HUNTER))
 
 def register_to_exit(node):
-    id = node.id
-    os.system("ip netns exec netlab-{id} curl -XPOST 127.0.0.1:4877/settings -H 'Content-Type: application/json' -i -d '{data}'"
-              .format(id=id, data=json.dumps({"exit_client": EXIT_SELECT})))
+    os.system(("ip netns exec netlab-{} curl -XPOST " +
```

Comment:
Its safer to use `subprocess.call` or `subprocess.check_output` when you need to gather stdout.